pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -312,13 +301,30 @@ private static void start() {
         }
 
         post(API.REGISTRATION, new UserRegistrationRoute(interpreter), responseSerializer);
-
         post(API.TEMP_USERS, new CreateTemporaryUserRoute(), responseSerializer);
 
+        // Admin APIs
```

Comment:
Have a better feeling for how to layout the admin APIs, so I moved and renamed a bunch of things.